src/lib/Microsoft.Fx.Portability.Azure/Storage/AzureStorage.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Fx.Portability.ObjectModel;
+using Microsoft.WindowsAzure.Storage;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.Fx.Portability.Azure.Storage
+{
+    public class AzureStorage : IStorage
+    {
+        private readonly StorageBlob _blob;
+        private CloudStorageAccount _storageAccount;
+
+        public AzureStorage(CloudStorageAccount storageAccount)
+        {
+            _storageAccount = storageAccount;
```

Comment:
nit: add null check
Do we need to save `storageAccount` in a member variable? It does not appear to have any use afterwards.

Reply:
Good point! Will remove.

---
In reply to: [178096310](https://github.com/Microsoft/dotnet-apiport/pull/598#discussion_r178096310) [](ancestors = 178096310)

Reply:
Added null check

---
In reply to: [179245302](https://github.com/Microsoft/dotnet-apiport/pull/598#discussion_r179245302) [](ancestors = 179245302,178096310)